### ISSUE TITLE
Wrong task Event API in docs

### ISSUE THREAD

**Issue by gr2m** [hoodiehq-archive/hood.ie-website-old#58]
follow up for #56, related to https://github.com/hoodiehq/hoodie.js/issues/249

Totally my fault I think, there is something wrong with the documented task API.

``` js
// as documented today
hoodie.task.on('message:start:123',   function (newMessageTask, options) {});

// as it should be
hoodie.task.on('message:123:start',   function (newMessageTask, options) {});
```

The order has been changed a while ago. It's always `type:id:action` for both, `hoodie.store` & `hoodie.task` API

/cc @jzaefferer @tenmilestereo

**Comment by lewiscowper** [comment]
So is the type:id:action format applicable in the remote method too?

Specifically "hoodie.remote.on('add:note:uuid123',    function (newObject) {});"

Should this be rewritten with the uuids in the middle too?

I can do this now regardless.

**Comment by gr2m** [comment]
correct. It should be `hoodie.remote.on('note:uuid123:add', function (newObject) {});`. Across hoodie API, we now follow this convention. If you see any docs / comments stating otherwise, feel free to fix :) And thanks for all your help, Lewis!

**Comment by lewiscowper** [comment]
No problem. :)

**Comment by lewiscowper** [comment]
This has been fixed with #59 being merged. :+1:

**Comment by svnlto** [comment]
yeah, thanks! @tenmilestereo :)
